src/components/Ramais/modais/AddRamal.vue

```diff
@@ -7,51 +7,28 @@ defineProps({
     type: Boolean,
     default: false,
   },
+  item: {
+    type: Object,
+    default: () => ({}),
+  },
 });
-const form = reactive({
-  ramalUser: "",
-  sectorUser: "",
-  ramalNumber: "",
-});
-
-async function addRamal() {
-  try {
-    await runMutation(AddRamal, { data: form });
+async function addRamal(form: Object) {
+  const result = await runMutation(AddRamal, { data: form });
+  if (result !== undefined) {
     positiveNotify(t("notifications.success.createRamal"));
-    emits("reload");
-  } catch {
-    negativeNotify(t("notifications.fail.createRamal"));
+    return emits("reload");
   }
+  negativeNotify(t("notifications.fail.createRamal"));
 }
 </script>
 
 <template>
   <DynamicDialog
+    hideControls
```

Comment:
quando for passar uma prop sempre deixar em kebab-case, então no caso troca de `hideControls` para `hide-controls`